spec/web-server.spec.js

```diff
@@ -0,0 +1,259 @@
+describe("web-server", function () {
+    "use strict";
+
+    var fs, logger, restify, restMiddleware, webServer, WebServer;
+
+    WebServer = require("../lib/web-server");
+
+    beforeEach(function () {
+        fs = {
+            readFileSync: function () {}
+        };
+
+        logger = {
+            console: function () {},
+            debug: function () {},
+            error: function () {},
+            info: function () {},
+            warn: function () {}
+        };
+
+        restify = {
+            createServer: function () {
+                return {};
+            }
+        };
+
+        restMiddleware = function () {
+
+        };
+
+        spyOn(fs, "readFileSync");
+        spyOn(logger, "console");
```

Comment:
We should have a fake logger module that does the spying automatically for you.  `require("./mocks/logger-mock")` sort of thing.